src/oss/confluent.sh

```diff
@@ -1401,7 +1449,12 @@ shift
 case "${command}" in
     help)
         if [[ -n "${1}" ]]; then
-            command_exists "${1}" && ( "${1}"_usage || invalid_command "${1}" )
+            if command_exists "${1}" ; then
+                ( "${1}"_usage || invalid_command "${1}" )
+            fi
+            if dynamic_cli_command_exists "${1}" && dynamic_cli_file_exists "${1}"; then
```

Comment:
Again this condition is diverging more than it is necessary here. 

The `dynamic_cli_file_exists` should take place inside the thin wrapper `acl_command` (which seems more appropriate as a name for `dynamic_cli_command` given the naming of the other wrappers, such as `list_command`, `start_command`, etc).